scheduler/src/cook/mesos/scheduler.clj

```diff
@@ -1041,14 +1143,18 @@
     (try
       (let [jobs (->> (sort-jobs-by-dru filtered-db unfiltered-db)
                       ;; Apply the offensive job filter first before taking.
-                      offensive-job-filter)]
-        (log/debug "Total number of pending jobs is:" (count jobs)
-                   "The first 20 pending jobs:" (take 20 jobs))
+                      (map (fn [[category jobs]]
```

Comment:
I'm not sure that will reduce the amount of boilerplate, since it's important to see which category's being filtered (for debugging prod issues), and I don't think that the complexity of the operation merits hiding it (since it's a pretty simple transform).